assets/js/templates/sidebar-items.handlebars

```diff
@@ -1,50 +1,58 @@
 <ul id="full-list">
-{{#each nodes as |node nodeId|}}
+  {{#each nodes as |node nodeId|}}
     {{#newGroup ../this node.group}}
       <li class="group">{{node.group}}</li>
     {{/newGroup}}
-    <li class="{{#isLocal node.id}}clicked open{{/isLocal}}">
 
-    {{!-- When visiting a module page, the link to this module page
-    in the menu should not link to a new page, instead should link
-    to the top of the page itself. --}}
-    {{#isEmptyNode node}}
```

Comment:
I believe you have accidentally removed this functionality in your PR. You can see this in Timex docs themselves, for the module `Timex.Parse.ZoneInfo.Parser.Zone`, it should never expand in the menu, it should directly go to the page because it only has a "Top".

Reply:
Nevermind. I see you just implemented it differently. :)